packages/schematics/angular/component/schema.json

```diff
@@ -48,6 +48,13 @@
       "alias": "t",
       "x-user-analytics": 10
     },
+    "standalone": {
+      "description": "Whether the generated component is standalone.",
+      "type": "boolean",
+      "default": false,
+      "alias": "sa",
+      "x-user-analytics": 10
```

Comment:
Is standalone component generation something we want to track with analytics?  If so 15 appears to be the only free slot currently.
/cc @mgechev